packages/mermaid/src/docs/community/code.md

```diff
@@ -0,0 +1,163 @@
+# Contributing Code
+
+The basic steps for contributing code are:
+
+```mermaid
+graph LR
+    git[1. Checkout a  git branch] --> codeTest[2. Write tests and code] --> doc[3. Update documentation] --> submit[4. Submit a PR] --> review[5. Review and merge]
+```
+
+1. **Create** and checkout a git branch and work on your code in the branch
+2. Write and update **tests** (unit and perhaps even integration (e2e) tests) (If you do TDD/BDD, the order might be different.)
+3. **Let users know** that things have changed or been added in the documents! This is often overlooked, but _critical_
+4. **Submit** your code as a _pull request_.
+5. Maintainers will **review** your code. If there are no changes necessary, the PR will be merged. Otherwise, make the requested changes and repeat.
+
+## 1. Checkout a git branch
+
+Mermaid uses a [Git Flow](https://guides.github.com/introduction/flow/)–inspired approach to branching.
+
+Development is done in the `develop` branch.
+
+Once development is done we create a `release/vX.X.X` branch from `develop` for testing.
+
+Once the release happens we add a tag to the `release` branch and merge it with `master`. The live product and on-line documentation are what is in the `master` branch.
+
+**All new work should be based on the `develop` branch.**
+
+**When you are ready to do work, always, ALWAYS:**
+
+1. Make sure you have the most up-to-date version of the `develop` branch. (fetch or pull to update it)
+2. Check out the `develop` branch
+3. Create a new branch for your work. Please name the branch following our naming convention below.
+
+We use the follow naming convention for branches:
+
+```text
+   [feature | bug | chore | docs]/[issue number]_[short description using dashes ('-') or underscores ('_') instead of spaces]
+```
+
```

Comment:
Is it worth adding a link to https://github.com/mermaid-js/mermaid/blob/develop/.github/pr-labeler.yml, which will show the current supported branch prefixes?

Reply:
Done